kube/objects.go

```diff
@@ -182,7 +190,16 @@ func (c *Client) FetchObjects(ctx context.Context, filter ObjectFilter) (*Object
 		err = annotateFetchError("CronJobs", err)
 		return
 	})
-
+	g.Go(func() (err error) {
```

Comment:
Personal preference would be to not use a named return parameter and instead return any errors explicitly for clarity's sake.

Reply:
The rest of the threads here use this paradigm - should I update all for consistency?

Reply:
Let's stick to the existing pattern then for now to be consistent. It's not a big thing anyway, and if we want to change it we can do it in one go for all usages through a follow up thread.